src/Module.h

```diff
@@ -141,6 +144,9 @@ class Module {
     /** Set the auto_schedule text for the Module. It is an error to call this
      * multiple times for a given Module. */
     EXPORT void set_auto_schedule(const std::string &auto_schedule);
+
+    /** Set whether this module uses strict floating-point directives anywhere. */
+    EXPORT void set_any_strict_float(bool any_strict_float);
```

Comment:
What happens if you call this inappropriately? Or, put another way: who should call this and when? (Seems like misusing it could produce weird and bad results.)

Reply:
Setting it wrong simply changes how optimization is done. Specifically it turns off global fast math, which might affect the runtime somehow. Turning it on inappropriately might make something slower, but I doubt it breaks anything. The failure mode would be not calling it when strict_float is used inside the Module.